libraries/botbuilder-applicationinsights/src/telemetryInitializerMiddleware.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+
+import { BotTelemetryClient, NullTelemetryClient, TelemetryLoggerMiddleware } from 'botbuilder-core';
+import { Middleware } from 'botbuilder-core';
+import { TurnContext } from 'botbuilder-core';
+import { TelemetryClient } from 'applicationinsights';
+import { CorrelationContext } from 'applicationinsights/out/AutoCollection/CorrelationContextManager';
+import * as appInsights from 'applicationinsights';
+
+/**
+ * Middleware for storing the incoming activity to be made available to Application Insights and optionally run the TelemetryLoggerMiddleware.
+ * Uses the botTelemetryClient interface.
+ */
+export class TelemetryInitializerMiddleware implements Middleware {
+
+    private readonly _telemetryClient: BotTelemetryClient;
+
+    // tslint:disable:variable-name
+    private readonly _logActivityTelemetry: boolean;
+    private readonly _logPersonalInformation: boolean;
+    // tslint:enable:variable-name
+
+    /**
+     * Initializes a new instance of the TelemetryInitializerMiddleware class.
+     * @param _telemetryClient The TelemetryClient used by the TelemetryLoggerMiddleware for logging activity telemetry.
+     * * @param _logActivityTelemetry (Optional) Enable/Disable logging of activity telemetry.
+     * * @param _logPersonalInformation (Optional) Enable/Disable logging original message name within Application Insights.
+     */
+    constructor(telemetryClient: TelemetryClient, logActivityTelemetry: boolean = false, logPersonalInformation: boolean = false) {
+        this._telemetryClient = telemetryClient || new NullTelemetryClient();
+        this._logActivityTelemetry = logActivityTelemetry;
```

Comment:
you used logActivityTelemetry, but did not use logPersonalInformation here.  you need to set that value into the class member as well.

Reply:
@benbrown You're right. Made me realize that we shouldn't have this param here anyway, otherwise you hide the ability for someone to set it on the underlying logging middleware.  Ill remove the logPersonalInformation from here and raise a PR in C# too to amend.

Reply:
@benbrown This param shouldn't be here and is a result of not passing in the logger middleware. I'll sort both tomorrow.

Reply:
@benbrown Removed the redundant param now I am passing in the logging middleware